runtime/doc/lua.txt

```diff
@@ -1073,16 +1070,20 @@ vim.ui_attach({ns}, {options}, {callback})                   *vim.ui_attach()*
 <
 
     Parameters: ~
-      • {ns}        (`integer`)
-      • {options}   (`table<string, any>`)
-      • {callback}  (`fun()`)
+      • {ns}        (`integer`) Namespace ID
+      • {opts}      (`table<string, any>`) Optional parameters:
+                    • {ext_… |ui-option|}? (`boolean`) If true, enable
+                      events for the respective UI element.
+                    • {cmdheight}? (`boolean`) If false, avoid setting
+                      'cmdheight' to 0 when `ext_messages` is enabled.
+      • {callback}  (`fun()`) function called for each UI event.
```

Comment:
Another (as well as am even separate PR) good moment to smuggle typing like in #27855 (runtime/lua/vim/_meta/builtin.lua) to reduce LSP warnings :wink: